app/controllers/controlled_work_document_selections_controller.rb

```diff
@@ -1,20 +1,19 @@
 class ControlledWorkDocumentSelectionsController < ApplicationController
   def new
     track_page_view(page: :cw_form_selection)
-    @form = ControlledWorkDocumentSelection.new
+    @form = ControlledWorkDocumentSelection.from_session(session_data)
     @check = Check.new(session_data)
     @model = CalculationResult.new(session_data)
   end
 
   def create
-    @form = ControlledWorkDocumentSelection.new(params[:controlled_work_document_selection]&.permit(:form_type, :language))
+    @form = ControlledWorkDocumentSelection.from_params(params, session_data)
```

Comment:
So we need to be able to store the values chosen in the session so they're accessible later (in the downloads controller). So to do that we need to make this model `SessionPersistable`. This also allows us to re-populate the form if the user clicks 'back' from the end of journey screen. Since the model is now SessionPersistable, it's more concise and consitent to use `from_params` here than to have to manually specify which params to pull in.